doc/adr/0006-explicit-resources-for-declaring-service-bindings.md

```diff
@@ -0,0 +1,61 @@
+# 6. Explicit resources for declaring service bindings
+
+Date: 2024-01-16
+
+## Status
+
+Proposed
+
+Enables [7. Deprecate ServiceSync](0007-deprecate-servicesync.md)
+
+## Context
+
+Service bindings are the configurations that determine how a Skupper
```

Comment:
Sorry Paul, I just realised from your comment that this PR inadvertently included commits from other PRs. I;ve fixed that now, but your comment applies to #1358 I believe.

Am I right in understanding your point here being that it would be valuable to call out in the consequences that the decision will require extra declaration(s) as compared to 1.x? (If so, I agree, that would be a useful addition and I will make it).